PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Description
+Describe the goals of the PR.
+# Related PRs
+List related PRs with links.
+# To Dos
```

Comment:
Let's leave out the To Dos. We should have an expectation for a complete PR including tests and documentation. I can't think of anything else we'd regularly want in a To Do section.

Reply:
Makes sense. I made the change.